var/spack/repos/builtin/packages/eckit/package.py

```diff
@@ -135,6 +156,12 @@ def cmake_args(self):
             self.define("ENABLE_SANDBOX", False),
         ]
 
+        # Static build of eckit not working, many places in eckit's build
+        # system have SHARED hardcoded (in several CMakeLists.txt files).
+        if "~shared" in self.spec:
+            # args.append("-DBUILD_SHARED_LIBS=OFF")
+            raise InstallError("eckit static build not supported")
```

Comment:
Why adding a variant to raise an InstallError during the build? Can't we just hard-code that `BUILD_SHARED_LIBS` is `ON`?

Reply:
What do you think @climbfuji?

Reply:
Changing this is fine with me, either logic reminds us (me) that we have to implement this at some point.

Reply:
@alalazo Just pushed the requested changes.